chromium_src/components/search_engines/template_url_prepopulate_data.cc

```diff
@@ -186,6 +368,23 @@ GetBravePrepopulatedEnginesForCountryID(
 
 }  // namespace
 
+// Some engines (like Yahoo) have different URLs per country
+// The intention of this function is to find the generic one
+// (ex: PREPOPULATED_ENGINE_ID_YAHOO) and then substitute the
+// country specific version.
+void LocalizeEngineList(
+    std::vector<BravePrepopulatedEngineID>* engines,
+    int country_id) {
+  for (size_t i = 0; i < engines->size(); ++i) {
```

Comment:
for (auto& engine : engines)

Reply:
Since this would be replacing `engine`, I'm not sure that is a feasible way to iterate the container. Would we still be able to replace the value of `engine`?

Reply:
`engine` here is non-cost reference, so making changes to it would change the element in the container